src/range/view/range.jsx

```diff
@@ -27,7 +27,8 @@ function LowerSlider(props) {
             popupProps={{
                 visible: tooltipVisible,
                 onVisibleChange: onTooltipVisibleChange,
-                animation: tooltipAnimation
+                animation: tooltipAnimation,
+                needAdjust: false
```

Comment:
Did this change fix https://github.com/alibaba-fusion/next/issues/176 ? It is working when needAdjust is default(i.e. true) previously, so maybe this isn't the main reason? Also I think such change will make customers' existing layout change

Reply:
This is because when an element is not visible in the viewport it will try many combinations of alignments to make it visible. We do not want this for the range.